apps/flame-defi/package.json

```diff
@@ -66,11 +67,10 @@
     "@types/react": "^18.3.12",
     "@types/react-dom": "18.3.1",
     "@types/uuid": "^9.0.8",
-    "autoprefixer": "^10.4.20",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "postcss": "^8.4.49",
-    "tailwindcss": "^3.4.17",
+    "tailwindcss": "^4.0.9",
```

Comment:
Should we move this to `dependencies` while we're updating this dep? I think it makes more sense there, even though everything is technically a dev dependency. I don't have a super strong opinion about it though